### PR TITLE
feat: add Exodus Wallet

### PR DESCRIPTION
**Pull request by gutenye**
Screen Shot 2022-07-06 at 19 39 36

### CODE REVIEW

**Comment by nickbytes** [comment]
Thanks for the contribution here @gutenye – we're currently discussing out strategy for adding new wallets to the repo and how we can best serve the RainbowKit community while also not being gatekeepers/maintainers of every wallet integration. I'll let you know when we have an update! Thanks for your patience!

**Comment by ExodusAndy** [comment]
@nickbytes hope all is well - any update on this?

**Comment by markdalgleish** [comment]
Sorry for the delay on this, and thanks for the PR!

We've just re-worked RainbowKit internals in v0.7 so that wallets can be imported individually, ensuring our bundle size doesn't keep increasing as we continue to add more wallets.

Are you able to give us access to your fork so that we can get you updated to the new API and apply some changes before merging? Alternatively, I've already made the updates in a branch if you'd like to replay them manually yourself: https://github.com/rainbow-me/rainbowkit/tree/%40markdalgleish/exodus-pr-updates

**Comment by timlanahan** [comment]
@markdalgleish I rebased your branch onto the PR

**Comment by bulgakovk** [comment]
Hey there! Just rebased the PR again, please lemme know if there's something else missing

File: site/components/Provider/Provider.tsx
Comment (DanielSinclair) [inline site/components/Provider/Provider.tsx:38]: Please remove these changes to the docs site.

**Review by DanielSinclair — CHANGES_REQUESTED** [review]

File: site/components/Provider/Provider.tsx
Comment (bulgakovk) [reply site/components/Provider/Provider.tsx]: @DanielSinclair Done in https://github.com/rainbow-me/rainbowkit/pull/567/commits/e8458bea1471b745d33f885de242b6a30cf86413

**Comment by olistic** [comment]
@DanielSinclair Anything else we can do to help you guys getting this across the line?

**Comment by DanielSinclair** [comment]
Seeing a few performance issues on our example dApp deployment here: https://rainbowkit-example-4jvxs45am-rainbowdotme.vercel.app/
When refreshing, the injected connector is not always detected; this pushes users to the install flow and prevents auto reconnect on dApps they've already interacted with. The likely cause is a race condition between RainbowKit initialization and the extension performing additional work before injecting the connector.

**Comment by DanielSinclair** [comment]
Closing for now, but will revisit and reopen if this connector begins working as expected

**Pull request closed without merging**
